<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multichannel Analog-to-Digital USB-converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,3,1">
    <item row="1" column="2">
     <widget class="QGroupBox" name="gbGUICfg">
      <property name="title">
       <string>GUI config</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>T offset</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="dsbVRef">
         <property name="suffix">
          <string>V</string>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>3.300000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Vref</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="hsTOffset_GUI">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>T scale</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="gbDump">
         <property name="title">
          <string>Dump data</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLineEdit" name="leDumpPath">
            <property name="text">
             <string>./adc.log</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pbDoDump">
            <property name="text">
             <string>Dump</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cbVScale">
         <item>
          <property name="text">
           <string>4 V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>400 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>200 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>40 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20 mV</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>V offset</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="hsVOffset_GUI">
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cbTScale">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>100 us</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>200 us</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500 us</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20 ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50 ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100 ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>200 ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500 ms</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 sec</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 sec</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 sec</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>V scale</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lPlot">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="gbDeviceCfg">
      <property name="title">
       <string>Device config</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="9" column="0" colspan="2">
        <widget class="QGroupBox" name="gbTrigger">
         <property name="title">
          <string>Trigger</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="2" column="1">
           <widget class="QSlider" name="hsTrigLevel">
            <property name="maximum">
             <number>4095</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cbTrigChannel"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>T offset</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>level</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="dsbTrigTMin">
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="dsbTrigOffset">
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-9999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QComboBox" name="cbTrigger">
            <item>
             <property name="text">
              <string>NONE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>RISING</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FALLING</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>THRESHOLD</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>STROBE LO</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>STROBE HI</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Strobe min</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>channel</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Strobe max</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="dsbTrigTMax">
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="pbOnce">
         <property name="text">
          <string>Start once</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="gbChannels">
         <property name="title">
          <string>Channels</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>samples</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>gain</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSlider" name="hsGain">
         <property name="maximum">
          <number>11</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="cbSamples">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>1024</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2048</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4096</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8192</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16384</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32768</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>65536</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>131072</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>262144</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>524288</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1048576</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="pbContinuous">
         <property name="text">
          <string>Continuous</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSlider" name="hsOffset">
         <property name="maximum">
          <number>4095</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="cbFrequency">
         <item>
          <property name="text">
           <string>STOPPED</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MAX</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500 kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>200 kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100 kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50 kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20 kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 kHz</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 kHz</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>freq</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>offset</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cbNBits">
         <item>
          <property name="text">
           <string>DIGITAL (2)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LO (4)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MID (8)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HI (12)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>bits</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QFormLayout" name="formLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
   </widget>
   <addaction name="menuDevice"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
